resources/production/chainspec.toml

```diff
@@ -29,6 +29,14 @@ auction_delay = 3
 # The delay for the payout of funds, in eras. If a withdraw request is included in a block in era N
 # (other than the last one), they are paid out in the last block of era N + locked_funds_period.
 locked_funds_period = 15
+# Round seigniorage rate represented as a fractional number
+#
+# Annual issuance: 2%
+# Minimum round exponent: 14
+# Ticks per year: 31536000000
+#
+# (1+0.02)^((2^14)/31536000000)-1 is expressed as a fraction below.
+round_seigniorage_rate = [6_414, 623_437_335_209]
```

Comment:
But in this chainspec the minimum round exponent is 17, not 14. Would be good to keep these values in sync.